kanidmd/idm/src/macros.rs

```diff
@@ -168,7 +168,7 @@ macro_rules! run_idm_test {
     }};
 }
 
-#[cfg(test)]
+// #[cfg(test)]
```

Comment:
Oh, are these used somewhere else?

Reply:
Yep, in `cargo bench`

Reply:
Ahhhh okay. Maybe they should be #[cfg(test, bench)] ?

Reply:
that doesn't seem to be an option as far as I can tell? :(